### PR TITLE
Test/add tests

### PR DESCRIPTION
**Pull request by adel-gu**
# Implemented changes in this PR:

- Implement tests for all components.
- Implemented tests for the logic files, both `calculate.js` and `operate.js`.

### CODE REVIEW

File: src/__tests__/Header.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from '../components/Header';
+
+describe('Testing the Header component:', () => {
+  test('should renerd correctly', () => {
+    render(<Header />, { wrapper: BrowserRouter });
+    expect(screen).toMatchSnapshot();
+  });
+});
```
Comment (GzimAsani) [inline src/__tests__/Header.test.js:10]: - [x] You need to either remove or add passing tests please
image

File: src/__tests__/Quote.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react';
+import Quote from '../pages/Quote';
+
+describe('Testing the Quote page:', () => {
+  test('should renerd correctly', () => {
+    render(<Quote />);
+    expect(screen).toMatchSnapshot();
+  });
+});
```
Comment (GzimAsani) [inline src/__tests__/Quote.test.js:9]: - [x] Good job adding tests for Quote, Home etc. But you need to cover all the functionalities & scenarios of the operate.js and calculate.js modules

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @adel-gu  and @SelmaNdapanda                 :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job adding snapshots and tests for Home and Quote:+1:  :+1: 

 ## Required Changes 


- [x] On readme file kindly add how can the users test your app(commands to run)
- [x] Check the comments below :arrow_double_down: 







 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/Header.test.js (hunk 1 above)
Comment (adel-gu) [reply src/__tests__/Header.test.js:10]: Hi @GzimAsani 👋

Thank you for mentioning this, I was wondering what should I do with it the whole time 😅.
I will do so 😎

File: src/__tests__/Quote.test.js (hunk 2 above)
Comment (adel-gu) [reply src/__tests__/Quote.test.js:9]: Thank you @GzimAsani , I'll definitely work on that 💪

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @adel-gu 👋, 

Well done for implementing all the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adel-gu** [comment]
Thank you @codepantha 👏👏👏, that's a funny GIF 😂😂😂

**Comment by codepantha** [comment]
Lol. I like it very much 😆

**Pull request merged**